apps/minifront/src/state/ibc-out.ts

```diff
@@ -104,15 +105,7 @@ export const currentTimePlusTwoDaysRounded = (currentTimeMs: number): bigint =>
   return roundedTimeoutNs;
 };
 
-// Reference in core: https://github.com/penumbra-zone/penumbra/blob/1376d4b4f47f44bcc82e8bbdf18262942edf461e/crates/bin/pcli/src/command/tx.rs#L998-L1050
-const getTimeout = async (
-  chain: Chain,
-): Promise<{ timeoutTime: bigint; timeoutHeight: Height }> => {
-  const { channel } = await ibcChannelClient.channel({
-    portId: 'transfer',
-    channelId: chain.ibcChannel,
-  });
-
+const clientStateForChannel = async (channel?: Channel): Promise<ClientState> => {
```

Comment:
Previously created to share code between ibc-in and out. However, turns out ibc-in can now retrieve counterparty id from registry directly. Though, splitting this out to a new function is still not a bad thing as getting client state is kinda independent.